brjs-core/src/main/java/org/bladerunnerjs/model/AbstractJsLib.java

```diff
@@ -42,6 +42,18 @@ public AbstractJsLib(RootNode rootNode, Node parent, MemoizedFile dir)
 	}
 	
 	@Override
+	public List<LinkedAsset> modelSeedAssets() 
```

Comment:
`modelSeedAssets` doesn't feel right to me, we also don't talk about 'model' seeds in other methods. `seedAssets` may be better here.